README.md

```diff
@@ -1,5 +1,32 @@
 # randomUUID
 
 Stub project for polyfilling `randomUUID` as proposed for standardization in
-https://github.com/WICG/uuid and implemented in Node.js v15.6.0
-https://github.com/nodejs/node/pull/36729
+https://github.com/WICG/uuid and implemented in
+[Node.js v15.6.0](https://github.com/nodejs/node/pull/36729).
+
+## Usage
+
+Just `require()` this package as soon as possible in your project, so its
+functionality is available for other modules:
+
+```js
+require('randomuuid')
+```
+
+it will automatically detect and apply the polyfill functions in place, both in
+Node.js and browser code.
+
+TODO: generate ESM version, so it can be directly importable by browsers, or
+better than that, confirm it's possible to write the polyfill directly on it.
+
```

Comment:
If ESM version won't be available soon, create an issue for it and link to that issue from here.
```suggestion
NOTE: ESM `import`able version not yet available.  See [this issue](appropriate link) for status.
```